sdk/identity/identity/test/node/vsCodeCredential.spec.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import assert from "assert";
+import { createSandbox } from "sinon";
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+import { getTenantIdFromVSCode } from "../../src/credentials/vscodeCredential";
+
+describe("VSCodeCredential", function() {
+  it("getTenantIdFromVSCode should load the tenant on different OSs", function() {
```

Comment:
The idea here being that since we run our tests in different environments, this will already satisfy that by passing every time? It's just a sketch though.